lnrpc/lightning.pb.go

```diff
@@ -7,10 +7,11 @@
 package lnrpc
 
 import (
-	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
-	protoimpl "google.golang.org/protobuf/runtime/protoimpl"
 	reflect "reflect"
 	sync "sync"
+
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
```

Comment:
Hmm, did you format this file manually? The CI complains about this being different from just running `make rpc`.

Reply:
Not sure how that happened, I recompiled and it disappeared from the diff.